src/mantine-core/src/components/Slider/Slider/Slider.tsx

```diff
@@ -118,8 +118,14 @@ export const Slider = forwardRef<HTMLDivElement, SliderProps>(
     const position = getPosition({ value: _value, min, max });
     const _label = typeof label === 'function' ? label(_value) : label;
 
+    const minMaxStepRef = useRef<{ min: number; max: number; step: number }>({ min, max, step });
+    minMaxStepRef.current = { min, max, step };
```

Comment:
I figured that adding `useEffect` for one simple assignment would only slow things down, so I left it out